less_users/tests/test_event_done.py

```diff
@@ -0,0 +1,150 @@
+from datetime import date, timedelta
+
+import pytest
+
+from django.urls import reverse
+
+from freezegun import freeze_time
+
+from less_users.models import UserChallenge, Log
+
+
+@pytest.mark.django_db
+def test_view_events_unauthorised_follow(client):
```

Comment:
A w ogóle poprawiłaś, żeby użytkownik musiał być zalogowany?

Reply:
not yet. Przy widokach klasowych korzystałam z LoginRequiredMixin; jeśli tu mam zwykły funkcyjny widok, to wystarczy zrobić to przez `if request.user.is_authenticated / else...`?

Reply:
https://docs.djangoproject.com/en/4.1/topics/auth/default/#the-loginrequiredmixin-mixin
https://docs.djangoproject.com/en/4.1/topics/auth/default/#the-login-required-decorator

Reply:
o, albo mogę skorzystać z dekoratora `@login_required` :)

Reply:
Znalazłam też wczoraj ten dekorator, chyba w podobnym czasie co mi podesłałaś linki. To mi pasuje :)

Reply:
użyłam @login_required dla `def view_events(request)` 🏅  i to wczoraj, ale nie wiem czemu znów nie spushowały mi się zmiany na githuba ?!

Reply:
Jest spora szansa, że ich nie skomitowałaś i nie pushnęłaś. Albo skomitowałaś do złego brancha